app/lib/methods/helpers/mergeSubscriptionsRooms.ts

```diff
@@ -98,7 +99,7 @@ export const merge = (
 	mergedSubscription.blocker = !!mergedSubscription.blocker;
 	mergedSubscription.blocked = !!mergedSubscription.blocked;
 	mergedSubscription.hideMentionStatus = !!mergedSubscription.hideMentionStatus;
-
+	mergedSubscription.sanitizedFname = slugifyLikeString(mergedSubscription.fname || mergedSubscription.name);
```

Comment:
What's going to happen if both fname and name are null? Does `slugifyLikeString` with empty param return an error?

Reply:
the function will return an empty string